exporter/exporterhelper/common.go

```diff
@@ -158,15 +161,15 @@ type baseExporter struct {
 	convertResourceToTelemetry bool
 }
 
-func newBaseExporter(cfg config.Exporter, logger *zap.Logger, options ...Option) *baseExporter {
+func newBaseExporter(cfg config.Exporter, logger *zap.Logger, reqUnnmarshaller requestUnmarshaler, options ...Option) *baseExporter {
 	bs := fromOptions(options)
 	be := &baseExporter{
 		Component:                  componenthelper.New(bs.componentOptions...),
 		cfg:                        cfg,
 		convertResourceToTelemetry: bs.ResourceToTelemetrySettings.Enabled,
 	}
 
-	be.qrSender = newQueuedRetrySender(cfg.ID().String(), bs.QueueSettings, bs.RetrySettings, &timeoutSender{cfg: bs.TimeoutSettings}, logger)
+	be.qrSender = newQueuedRetrySender(cfg.ID().String(), bs.QueueSettings, bs.RetrySettings, reqUnnmarshaller, &timeoutSender{cfg: bs.TimeoutSettings}, logger)
```

Comment:
I'm not sure if that's possible but could we maybe change how `reqUnmarshaller` is passed in here/created?
Couldn't we use options here as well as they are used for other params (like `QueueSettings` or `RetrySettings` ?)